packages/block-library/src/site-logo/index.php

```diff
@@ -51,6 +51,22 @@ function render_block_core_site_logo( $attributes ) {
 	}
 
 	$classnames = array();
+
+	$border_attributes = block_core_site_logo_get_border_attributes( $attributes );
+	if ( $border_attributes ) {
+		$classnames[] = 'has-custom-border';
+
+		$processor = new WP_HTML_Tag_Processor( $custom_logo );
+		$processor->next_tag( 'img' );
+		if ( ! empty( $border_attributes['class'] ) ) {
+			$processor->set_attribute( 'class', $border_attributes['class'] );
```

Comment:
In case there are other classes already, should this use the processor's `add_class` method instead? Likewise could there be existing styles that need to be taken into account for the other `set_attribute` call?

Reply:
Good catch, this code removed the "custom-logo" class from the img on the front.